main/cmake/APKBUILD

```diff
@@ -41,6 +41,7 @@ build() {
 		--system-curl \
 		--system-expat \
 		--no-system-jsoncpp \
+		--no-system-librhash \
```

Comment:
Maybe move from community to main? Otherwise it will build embedded rhash

https://git.alpinelinux.org/cgit/aports/tree/community/rhash